src/Raven.Client/Documents/Operations/Replication/PullReplicationAsHub.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Raven.Client.Documents.Replication;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Replication
+{
+    public sealed class PullReplicationAsHub : ExternalReplication
+    {
+        public PullReplicationMode Mode = PullReplicationMode.HubToSink;
```

Comment:
this is not readonly, can it be changed?

Reply:
double check: should it be hubToSink or sinkToHub?

Reply:
> this is not readonly, can it be changed?

yes, if we'll use `PullReplicationAsHub` for client operations as mentioned above, we can override the `Mode`. For example, `Mode = PullReplicationMode.HubToSink | PullReplicationMode.SinkToHub;`.



> double check: should it be hubToSink or sinkToHub?

by default, we are using it as `HubToSink` because we are calling it from the outgoing hub connection.